l10n/en/firefox/whatsnew/whatsnew-donate.ftl

```diff
@@ -6,7 +6,11 @@
 
 whatsnew-donate-whatsnew = What’s New
 whatsnew-donate-donate-to = Donate to { -brand-name-mozilla-foundation }, the non-profit behind { -brand-name-firefox }
+whatsnew-donate-you-chose-v2 = You chose { -brand-name-firefox } because you know the internet should serve people, not power. Let’s take it further.
+# Obsolete string (expires 2026-04-04)
 whatsnew-donate-you-chose = You chose { -brand-name-firefox } because you know the Internet should serve people, not power. Let’s take it further.
+whatsnew-donate-you-can-help-v2 = You can help make good on the promise of the internet. Make an end-of-year donation to { -brand-name-mozilla-foundation } by Dec 31 to support people-first tech.
+# Obsolete string (expires 2026-04-04)
 whatsnew-donate-you-can-help = You can help make good on the promise of the Internet. Make an end-of-year donation to { -brand-name-mozilla-foundation } by Dec 31 to support people-first tech.
```

Comment:
QQ: Does this need versioning when the L10n PR was still quarantined, so none of these strings got exposed further?